frontend/cypress/integration/mainnet/mainnet.spec.ts

```diff
@@ -82,7 +82,7 @@ describe('Mainnet', () => {
                         cy.get('[ngbtooltip="Next Block"] > .ng-fa-icon > .svg-inline--fa').should('not.exist');
                         cy.get('[ngbtooltip="Previous Block"] > .ng-fa-icon > .svg-inline--fa').should('be.visible');
                         cy.document().left();
-                        cy.get('[ngbtooltip="Previous Block"] > .ng-fa-icon > .svg-inline--fa').should('be.visible');
+                        cy.get('h1').invoke('text').should('equal', 'Next block');
```

Comment:
@softsimon I would change this locator to `.title-block h1` as it's easier to identify what element it is just by looking at the test